tests/integration/components/bs-dropdown/menu/link-to-test.js

```diff
@@ -64,32 +60,30 @@ module('Integration | Component | bs-dropdown/menu/link-to', function (hooks) {
     });
   });
 
-  if (supportsAngleBracketsLinkTo) {
```

Comment:
This is checking `hasEmberVersion(3, 10)`, i.e. guards against Ember <3.10, which we don't support anymore.